### DISCUSSION TITLE
Toast Format question

### DISCUSSION THREAD

**Discussion by sgexcit** [Windos/BurntToast#142] [Q&A] [answered]
This may be due to a newer feature release of Windows 10, but BurntToast notifications now have a 'Windows PowerShell' at the top (or under an image if configured). Anyone know of a way to hide, modify or obfuscate this?

![image](https://user-images.githubusercontent.com/76213604/102619928-3421ad00-410b-11eb-947f-eba14563d7ff.png)

**Comment by Windos** [comment, marked as answer]
Unfortunately, there's no way to get rid of this. Microsoft wanted to make sure users can ID which applications are sending notifications, and it was too easy to miss this in the old location (at the bottom of the toast and without the icon).

You can change what it is by changing the AppID your toast is called by.

The way to do this on a semi-permanent basis is to edit the config.json file that ships with the module (but this will go back to default after a module update). n.b. if going this route pay attention to the default and note that things like slashes need to be doubled up.

You can also specify it at runtime if using `Submit-BTNotification` (e.g. `Submit-BTNotification -AppId 'AppID Here'`)

There's a PR open to make this available to change via `New-BurntToastNotification`/`toast` as well, but that'll be available in the next release.

Valid values for your AppID can be found by running `Get-StartApps`. It is possible to create fully custom ones, but it's an involved process and includes having to install a shim on each PC that'll display the toast.

**Comment by A9G-Data-Droid** [comment]
If your custom application doesn't already have a start menu item I have proposed a solution to this problem in [PR#168 ](https://github.com/Windos/BurntToast/pull/168)

It's a quick and easy way to make a custom application that you can send toast from. It creates a start menu item and registers the `AppId`.

**Reply by sgexcit** [reply]
I actually used the AppID of an application we use, and if it didn't detect that on the machine it defaulted to the Windows Settings AppID as a just-in-case but I appreciate that!

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
